src/modules/endpoint/sagas/loadTuNews.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import type { Saga } from 'redux-saga'
+import { createTunewsEndpoint, TunewsModel } from '@integreat-app/integreat-api-client'
+import { call } from 'redux-saga/effects'
+import { tunewsApiUrl } from '../constants'
+
```

Comment:
i think we should use `loadTunews` instead of `loadTuNews` to be consistent with other places in the code